packages/firestore/test/integration/api/database.test.ts

```diff
@@ -514,4 +515,55 @@ apiDescribe('Database', persistence => {
       return Promise.resolve();
     });
   });
+
+  asyncIt('can queue writes while offline', () => {
+    return withTestDb(persistence, db => {
+      const docRef = db.collection('rooms').doc();
+      const firestoreClient = (docRef.firestore as Firestore)._firestoreClient;
+
+      return firestoreClient
+        .disableNetwork()
+        .then(() => {
+          return Promise.all([
+            docRef.delete(),
```

Comment:
Can you make the same change Gil did for iOS? (Sorry, I'm not awesome enough to do it for you)

Reply:
Ported the changes over.